src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,79 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
-
+        FileInputStream fis = null;
+        try {
+            fis = new FileInputStream(fromFileName);
+        } catch (FileNotFoundException e) {
+            throw new RuntimeException("Can't read data from the file", e);
+        }
+        byte[] buffer = new byte[10];
+        StringBuilder sb = new StringBuilder();
+        while (true) {
+            try {
+                if (fis.read(buffer) == -1) {
+                    break;
+                }
```

Comment:
Your method reads data from the input file in chunks of 10 bytes and appends it to a StringBuilder. However, this approach assumes that the file contains only ASCII characters. If the file contains multibyte characters (e.g., UTF-8 encoded text), this method may break the characters and produce incorrect results. Look at the [this](https://mate.academy/learn/java-core/java-builder-and-files?section=video&videoId=1795) video to understand better approach to read a data from file.